week-4/E-arrays-of-objects/exercise-1.js

```diff
@@ -5,42 +5,65 @@
 */
 
 var person1 = {
-    name: "Alice",
-    age: 25
+  name: "Alice",
+  age: 25
 };
 
 var person2 = {
-    name: "Bob",
-    age: 30
+  name: "Bob",
+  age: 30
 };
 
 var person3 = {
-    name: "John",
-    age: 20
+  name: "John",
+  age: 20
 };
 
 /* 
 DO NOT EDIT ANYTHING ABOVE THIS LINE
 WRITE YOUR CODE BELOW
 */
 
+var persons = [person1, person2, person3]; // Complete here
 
-var persons = // Complete here
+var personNames = persons.map(getName);
 
-var personNames = // Complete here
+function getName(persons) {
+  return persons.name;
+}
 
-var personsYoungerThan28YearsOld = // Complete here
+//[person1.name, person2.name, person3.name]; // Complete here
 
+var personsYoungerThan28YearsOld = persons.filter(checkingAge); // Complete here
+
+function checkingAge(persons) {
+  return persons.age <= 28;
+}
```

Comment:
*Younger than*.  28 is not younger than 28...  Always be careful with inequalities and check whether they're inclusive or exclusive.  Many bugs hide here.